extensions/grpc/cli/src/main/resources/application.properties

```diff
@@ -0,0 +1,4 @@
+quarkus.log.level=INFO
+quarkus.banner.enabled=false
+quarkus.package.jar.type=uber-jar
+quarkus.package.jar.add-runner-suffix=false
```

Comment:
This won't work I believe.
When building Quarkus from scratch, you won't find the plugin, because the plugin is built. 

In another case, I had to switch to the assembly maven plugin (would love to switch back to the Quarkus plugin).

Reply:
Hmm, yeah, this could be a problem ... since my copied example was from Quarkiverse, hence made it more easy.
Any ideas / suggestions what can be done?

Reply:
Yes :-) I'm going to push a PR today that does this.

Reply:
is there a reason we do uberjar here? why not just reuse users most likely already downloaded deps for this cli?

Reply:
@maxandersen seems like Uber-jar is the way to distribute self-contained commands. At least that's what I got from the doc and when I tried the "fast-jar" approach, it did not work (and anyway, fast jar would need quarkus before quarkus)

Reply:
yeah okey - if the actual cli is a quarkus app and not "just" a java app you need it. It would be interesting if we had a way to make a jar that has only the classes that need changing and rest could just be fetched using dependencies.

Reply:
hmm - even uber-jar would need quarkus before quarkus would it not?

Reply:
Using Quarkus for these commands works, but yes it might be overkill. 

By using the assembly plugin it works because it waits for all dependencies to be built before building that fat jar. Having a plugin dependency is slightly more complicated and fail in all my attempts from scratch. Funny, I had the same issue more than 10 years ago where I had to have profiles to build the plugin and then the modules using the plugin.

Reply:
We should be able to build the Maven plugin right after core-deployment to make it available for other extensions.